### DISCUSSION TITLE
New Images for .NET Framework 4.8.1

### DISCUSSION THREAD

**Discussion by mthalman** [microsoft/dotnet-framework-docker#1005] [Announcements]
# New images for .NET Framework 4.8.1

New images have been released with support for [.NET Framework 4.8.1 runtime and SDK](https://devblogs.microsoft.com/dotnet/announcing-dotnet-framework-481/).

## Details

* https://github.com/microsoft/dotnet-framework-docker/pull/996

You can access these new images with the following tags:

* `mcr.microsoft.com/dotnet/framework/aspnet:4.8.1`
* `mcr.microsoft.com/dotnet/framework/wcf:4.8.1`
* `mcr.microsoft.com/dotnet/framework/runtime:4.8.1`
* `mcr.microsoft.com/dotnet/framework/sdk:4.8.1`

Check out the [Docker Hub page](https://hub.docker.com/_/microsoft-dotnet-framework) for more information on the new tags.

The .NET Framework 4.8.1 runtime is only supported on Windows Server 2022. If you attempt to pull the `4.8.1` tag from an earlier version of Windows Server, you will get an error like `a Windows version 10.0.20348.887-based image is incompatible with a 10.0.XXXXX host` because the container will not be [compatible with that OS version](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility).

The 3.5 and 4.8 SDK images have also been updated to contain the .NET Framework 4.8.1 targeting pack. This allows those images to be used to build .NET Framework projects that target 4.8.1. Note that this only allows for the projects to be built in that container. It won't be possible to test or otherwise run a 4.8.1 application in that environment. You would need one of the 4.8.1 images for that.

**Comment by richlander** [comment]
The following container improvements are in the 4.8.1 images: https://github.com/microsoft/dotnet-framework-docker/discussions/935
